sound/soc/sof/ipc4-topology.h

```diff
@@ -349,6 +349,23 @@ struct sof_ipc4_src {
 	struct sof_ipc4_msg msg;
 };
 
+/**
+ * struct sof_ipc4_base_module_cfg_ext - base module config extension containing the pin format
```

Comment:
commit message:
This will be appended to the init instance IPC payload for modules that require it.

How do we know which modules require this module_cfg_ext?